core/domain/wipeout_jobs_one_off.py

```diff
@@ -39,14 +38,10 @@ def map(pending_deletion_request_model):
         """Implements the map function for this job."""
         pending_deletion_request = wipeout_service.get_pending_deletion_request(
             pending_deletion_request_model.id)
-        if pending_deletion_request.deletion_complete:
-            yield ('ALREADY DONE', pending_deletion_request.user_id)
-        else:
-            wipeout_service.delete_user(pending_deletion_request)
-            pending_deletion_request.deletion_complete = True
-            wipeout_service.save_pending_deletion_requests(
-                [pending_deletion_request])
-            yield ('SUCCESS', pending_deletion_request.user_id)
+        yield (
+            wipeout_service.run_user_deletion(pending_deletion_request),
```

Comment:
It's not clear what the return value of this would be. Perhaps declare a variable to explain that and yield that variable instead.

Reply:
Done.